evalml/pipelines/components/utils.py

```diff
@@ -274,6 +281,13 @@ def generate_component_code(element):
 
     Raises:
         ValueError: If the input element is not a component instance.
+
+    Examples:
+        >>> from evalml.pipelines.components.estimators.regressors.decision_tree_regressor import DecisionTreeRegressor
+        >>> assert generate_component_code(DecisionTreeRegressor()) == "from evalml.pipelines.components.estimators.regressors.decision_tree_regressor import DecisionTreeRegressor\n\ndecisionTreeRegressor = DecisionTreeRegressor(**{'criterion': 'mse', 'max_features': 'auto', 'max_depth': 6, 'min_samples_split': 2, 'min_weight_fraction_leaf': 0.0})"
+        ...
```

Comment:
Docs don't build without the `r` prefix? Is it cause of these three dots here? The example renders fine in the doc so I'm just wondering what happened but no need to change anything.

Reply:
The lint check doesn't pass, I think due to the `\n` characters in the output